lib/MetaCPAN/Server/Controller/Favorite.pm

```diff
@@ -45,5 +45,12 @@ sub recent : Path('recent') : Args(0) {
     $c->stash($data);
 }
 
+sub leaderboard : Path('leaderboard') : Args(0) {
+    my ( $self, $c ) = @_;
+    my $data = $self->model($c)->raw->leaderboard();
+    $data or return;
+    $c->stash($data);
```

Comment:
In general, I wonder if we should just shorten these to:

`return $c->stash( $self->model($c)->raw->leaderboard || {} );`

Not sure if you can stash an `undef`.